deepcell/image_generators/__init__.py

```diff
@@ -254,6 +254,9 @@ def _transform_masks(y, transform, data_format=None, **kwargs):
 
 from deepcell.image_generators.tracking import SiameseDataGenerator
 from deepcell.image_generators.tracking import SiameseIterator
+
+from deepcell.image_generators.cropping import CroppingDataGenerator
+from deepcell.image_generators.cropping import CropperIterator
```

Comment:
The prefixes for these two are different (`Cropping` and `Cropper`). Let's make them the same name, for easier importing etc.